core/config/config.go

```diff
@@ -0,0 +1,91 @@
+package config
+
+import (
+	"fmt"
+
+	"github.com/spf13/viper"
+)
+
+// File defines the fields required to point to a local config file
+type File struct {
```

Comment:
Too generic name. Would this be used for plugins?

Reply:
> Would this be used for plugins?

No, since the local config file will be present in the host CLI environment, this will we used over there.

Resolved via commit: https://github.com/aerogear/charmil/pull/108/commits/1eb64335a56861836a41f76cbbc5e6c7becf119f